src/routes/rooms.ts

```diff
@@ -67,7 +75,6 @@ router.post(
     });
     const errors = await validate(room);
     if (errors.length > 0) {
```

Comment:
This check feels weird, can we be more specific since the exception is really really concrete?

Reply:
here we are checking if there are validation errors, can you give me an example of the level of specificity you were expecting? I'm not totally sure  what you mean 😱

Reply:
I feel like you are ```inferring``` a little bit here because you are using the Array state to throw an exception of an error that has nothing to do with the ```Array.length``` itself.

I would destructure the array, get the actual error, and if that error exist throw the exception. (in my opinion that would be more idiomatic)

Reply:
This actually could mean that you need a new API: ```hasErrors```.
In that way you can describe your process idiomatically :)

Reply:
Let me know what you think now 🙂